packages/components/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,93 @@
+import { Placement, flip, offset, shift, useFloating } from "@floating-ui/react-native";
+import React, { createContext } from "react";
+import { Animated, Modal, Pressable, View, ViewProps } from "react-native";
+import { EDSStyleSheet } from "../../styling";
+import { Paper } from "../Paper";
+import { useStyles } from "../../hooks/useStyles";
+
+export type MenuProps = {
```

Comment:
This can be set to be equal to PopoverProps.
`export type MenuProps = PopoverProps;`

Reply:
After updating PopoverProps to use the Placement type.